testsuite/forge/src/backend/k8s/helm-values/aptos-node-default-values.yaml

```diff
@@ -28,5 +28,12 @@ service:
     enableRestApi: true
     enableMetricsPort: true
 
+# to give HAProxy access to more resources, schedule it on validators nodegroup
+# in the long term, we should consider giving the "utilities" nodegroup more resources
+# or removing it from the reference implementation entirely
+haproxy:
+  nodeSelector:
+    eks.amazonaws.com/nodegroup: "validators"
```

Comment:
we don't need this change anymore, now that the `utilities` nodepool has been updated to 8vcpu 32gb ram https://github.com/aptos-labs/aptos-core/pull/4404

Reply:
removed